docs/design/coreclr/botr/ilc-architecture.md

```diff
@@ -129,7 +129,7 @@ Note that while "constructed" and "unconstructed" type nodes are modelled separa
 
 Related compiler switches: `--dgmllog` serializes the dependency graph into an XML file. The XML file captures all the nodes in the graph, but only captures the first edge leading to the node (knowing the first edge is enough for most purposes). `--fulllog` generates an even bigger XML file that captures all the edges.
 
-Related tools: [Dependency analysis viewer](../how-to-debug-compiler-dependency-analysis.md) is a tool that listens to ETW events generated by all the ILC compiler processes on the machine and lets you interactively explore the graph.
+Related tools: [Dependency analysis viewer](../../../../src/coreclr/tools/aot/DependencyGraphViewer/README.md) is a tool that listens to ETW events generated by all the ILC compiler processes on the machine and lets you interactively explore the graph.
```

Comment:
Looks like the file this was pointing to was renamed to debugging-compiler-dependency-analysis.md, but I agree it's better to point to the tool directly since this talks about the tool.